### ISSUE TITLE
Strange looping issue with IE9

### ISSUE THREAD

**Issue by theremon** [HubSpot/offline#31]
Hi guys and first of all thanks for this nice script.

Strangely, even tough everything works fine on Chrome, FF, IE10 & IE11, when I try to open my site using IE9 it goes into a reconnecting loop (which starts normally with the disconnected message), downloads the image I've set up fine (IE returns a proper 200 status), but all I get is the attempting to reconnect loop forever. Same goes with the settings I've given and with the default ones.
Unfortunately I'm not a Coffeescript guy so I can't mess with the code, so I'm just asking for ideas here...
I'll try to debug using the compiled JS files (God help me debug on IE). I know that I'll burn in hell or something for even mentioning IE9, but hey... customers still use it, so we gotta play along.

Thanks in advance for any help you can provide.

**Comment by indolering** [comment]
Given the age of this ticket and the fact that IE8 and IE9 have about 3.5% of the market share, I move to close this unless the the author responds with more information.

**Comment by theremon** [comment]
Hello again,

I'm terribly sorry for never replying to this thread. Should have done it quite some time ago.
The problem does not accur anymore. As I didn't check with every update (support for IE9 is more or less dropped on our apps), I can't say which version of Offline changed this behaviour (or maybe which revision to our code) but I cannot reproduce it anymore. Internet Explorer 9 currently works fine. I also upgraded to the latest version of Offline and confirmed again that the issue has been fixed.

Once again, sorry for leaving this open and thanks for still paying attention.
Consider the issue closed.
